enhancements/console/custom-logos.md

```diff
@@ -0,0 +1,206 @@
+---
+title: custom-logos
+authors:
+  - "@cajieh"
+reviewers:
+  - "@jhadvig "
+  - "@spadgett"
+  - "@everettraven"
+  - "@JoelSpeed"
+approvers:
+  - "@spadgett"
+api-approvers:
+  - "@JoelSpeed"
+creation-date: 2025-02-11
+last-updated: 2025-02-11
+---
+
+# Custom Logos
+
+## Summary
+
+The OpenShift Container Platform (OCP) web console was upgraded to PatternFly 6. In PatternFly 6, the masthead color changes based on the mode (light or dark). As a result, a single custom logo may not be suitable for both theme modes. 
+
+Add the ability to specify custom logos to support light and dark theme modes for the masthead and favicon.
+
+## Background info
+
+The OpenShift Container Platform (OCP) web console was upgraded to PatternFly 6. In PatternFly 6, the masthead color changes based on the mode (light or dark). As a result, a single custom logo may not be suitable for both theme modes. 
+
+To address this, we need to allow users to add custom logos for both the masthead and favicon that are compatible with light and dark themes. This ensures that the logos are always visible and aesthetically pleasing, regardless of the theme mode.
+
+The custom logos feature will enable users to specify different logos for the masthead and favicon based on the theme mode. This will involve exposing a new API endpoint to support custom logos for both light and dark themes.
+
+## Motivation
+
+The existing OKD and OpenShift logos are designed for a dark masthead background and include white text, making them less effective in a light theme. To ensure logos remain visible and visually appealing in both light and dark themes, users need the ability to add custom logos for the masthead and favicon that adapt to the theme mode. To support this, a new API endpoint will be introduced, allowing users to specify different logos for light and dark themes.
+
+### User Stories
+
+* As an OpenShift web console user, I want to be able to add different custom logos for light and dark theme modes in the masthead and favicon.
+
+### Goals
+
+This feature should allow users to add custom logos for the masthead and favicon that are compatible with both light and dark theme modes in the OpenShift web console.
```

Comment:
```suggestion
This feature will allow users to add custom logos for the masthead and favicon that are compatible with both light and dark themes in the OpenShift web console.
```
I would say use will instead of should since this is the intended outcome.